crates/symbolicator-service/src/types/objects.rs

```diff
@@ -229,6 +222,20 @@ impl AllObjectCandidates {
             }
         }
     }
+
+    /// Returns the vector of `ObjectCandidate`s backing this collection.
+    pub fn into_inner(self) -> Vec<ObjectCandidate> {
+        self.0
+    }
+}
+
+impl From<Vec<ObjectCandidate>> for AllObjectCandidates {
+    fn from(mut source: Vec<ObjectCandidate>) -> Self {
+        source
+            .sort_by_cached_key(|candidate| (candidate.source.clone(), candidate.location.clone()));
+        source.dedup_by_key(|candidate| (candidate.source.clone(), candidate.location.clone()));
```

Comment:
do we really need to clone here?

Reply:
As far as I can tell, we do, unfortunately. Just lifetime things.